code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -490,15 +490,18 @@
 			"<span class='userdanger'>[src] is trying to inject you!</span>")
 
 		var/failed = FALSE
-		if(do_mob(src, patient, 30))	//Is C == patient? This is so confusing
+		if(do_mob(src, patient, 30))
 			if((get_dist(src, patient) <= 1) && (on) && assess_patient(patient))
 				if(reagent_id == "internal_beaker")
 					if(use_beaker && reagent_glass && reagent_glass.reagents.total_volume)
 						var/fraction = min(injection_amount/reagent_glass.reagents.total_volume, 1)
+						var/reagentlist = pretty_string_from_reagent_list(reagent_glass.reagents.reagent_list)
+						log_combat(src, patient, "injected", "beaker source", "[reagentlist]:[injection_amount]")

```

Comment:
`"beaker source"`
log_combat expects an atom for that param. move it to the additional.

Reply:
it expects an atom, but doesn't actually need one

Reply:
have you tested it? iirc this results in the message just saying "no key"

https://github.com/tgstation/tgstation/blob/49a4cb67694a1f36c36fa016c9686721d11b4075/code/game/atoms.dm#L688
https://github.com/tgstation/tgstation/blob/49a4cb67694a1f36c36fa016c9686721d11b4075/code/game/atoms.dm#L696-L697

https://github.com/tgstation/tgstation/blob/eac76e84eae260c85c7c99e17e70964ad3a96711/code/__HELPERS/_logging.dm#L253

Reply:
I removed the key name because it was stupid, most things you are hit with are not mobs

Reply:
and it was putting a [DC} on all objects